Documentation/Functions/If.rst

```diff
@@ -215,6 +215,108 @@ equals
       if.value.data = GETENV:REQUEST_METHOD
 
 
+..  index:: if; contains
+..  _if-contains:
+
+contains
+--------
+
+..  versionadded:: 12.1
+
+:aspect:`Property`
+    contains
+
+:aspect:`Data type`
+    value / :ref:`stdwrap`
+
+:aspect:`Description`
+    Returns true, if the content contains :typoscript:`value`.
+
+:aspect:`Example`
+    ..  code-block:: typoscript
+        :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+        :emphasize-lines: 11
+
+        # Add a span tag before the page title if the page title
+        # contains the string "media"
+        page.10 = TEXT
+        page.10 {
+            data = page:title
+            htmlSpecialChars = 1
+            prepend = TEXT
+            prepend {
+                value = <span class="icon-video"></span>
+                if.value.data = page:title
+                if.contains = Media
+            }
+            outerWrap = <h1>|</h1>
+        }
+
+
+..  index:: if; startsWith
+..  _if-startsWith:
+
+startsWith
```

Comment:
Should we order them alphabetically or is this a logical sorting regarding order of execution or some such?

Reply:
Hmm, I followed the order in the code like the other properties. But can sort it alphabetically in a follow-up. Order does not matter for "if" (like in stdWrap AFAIK) as all conditions have to evaluate to true.